lib/functions.js

```diff
@@ -9,7 +9,8 @@ function bind(el, attr, val, remove = true) {
     if (remove) {
         el.removeAttr(attr)
     }
-    el.attr('v-bind:'+attr, val)
+    if(attr === '') el.attr('v-bind'+attr, val);
```

Comment:
please use proper if () {} else {} and also check for undefined, null to be on the save side